src/include/bootloader/routines/dialogs.rb

```diff
@@ -128,7 +128,7 @@ def MainDialog
 
       )
       if ret != :back && ret != :abort && ret != :cancel
-        @return_tab = CWMTab.LastTab
+        @return_tab = CWMTab.LastTab || "tab"
         @return_tab = "installation" if @return_tab.include? "tab" # workaround different tab set for grub2
```

Comment:
no as tab is proposed as initial tag elsewhere in code. So different part of code propose as default return_tab  "tab" and here it is changed to grub2 default return tab..it is just relict of old multi-bootloader code that have here condition if `bootloader == "grub2" &&` which is now useless. It deserve more cleaning but it is not part of this pr and I hope it will be cleaned together with perl-bootloader support drop off